robosuite/scripts/collect_human_demonstrations.py

```diff
@@ -131,28 +132,34 @@ def gather_demonstrations_as_hdf5(directory, out_dir, env_info):
             states.extend(dic["states"])
             for ai in dic["action_infos"]:
                 actions.append(ai["actions"])
+            success = dic["successful"]
```

Comment:
probably should make this `success = success or dic["successful"]` because the success can take place in different npzs, and right now the flag will only remember the last npz's success in the iteration. I'm also not sure if this iteration even goes through the npz files in order of collection

Reply:
Addressed: b06c7b4f9650ceedc9d32820ee57edf2c96acc20